Source/Core/DolphinQt/Settings/GeneralPane.cpp

```diff
@@ -382,3 +366,101 @@ void GeneralPane::GenerateNewIdentity()
   message_box.exec();
 }
 #endif
+
+void GeneralPane::AddDescriptions()
+{
+  static constexpr char TR_DUALCORE_DESCRIPTION[] = QT_TR_NOOP(
+      "Runs the emulated CPU and GPU on separate threads. Improves Dolphin's performance, but can "
+      "result in various glitches and crashes."
+      "<br><br>This setting cannot be changed while emulation is active."
+      "<br><br><dolphin_emphasis>If unsure, leave this checked.</dolphin_emphasis>");
+  static constexpr char TR_CHEATS_DESCRIPTION[] = QT_TR_NOOP(
+      "Enables the use of AR and Gecko cheat codes which can be used to modify games' behavior. "
+      "These codes can be configured with the Cheats Manager in the Tools menu."
+      "<br><br>This setting cannot be changed while emulation is active."
+      "<br><br><dolphin_emphasis>If unsure, leave this unchecked.</dolphin_emphasis>");
+  static constexpr char TR_OVERRIDE_REGION_SETTINGS_DESCRIPTION[] =
+      QT_TR_NOOP("Lets you use languages and other region-related settings that the game may not "
+                 "be designed for. May cause various crashes and bugs."
+                 "<br><br>This setting cannot be changed while emulation is active."
+                 "<br><br><dolphin_emphasis>If unsure, leave this unchecked.</dolphin_emphasis>");
+  static constexpr char TR_AUTO_DISC_CHANGE_DESCRIPTION[] = QT_TR_NOOP(
+      "Automatically changes the game disc when requested by games with 2 discs. This feature "
```

Comment:
The "2" here is odd. IMO, it's best to keep it as just text for consistency. So, "two discs".